src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenDeconstructTests.cs

```diff
@@ -1095,6 +1090,122 @@ public void Deconstruct(out int a, out int b)
         }
 
         [Fact]
+        [WorkItem(18629, "https://github.com/dotnet/roslyn/issues/18629")]
+        public void ValueTupleNotRequiredIfReturnIsNotUsed()
```

Comment:
Same for constraint violations.

---
In reply to: [123336564](https://github.com/dotnet/roslyn/pull/20295#discussion_r123336564) [](ancestors = 123336564)